Apto/Plugins/PartsListElement/Application/Core/Service/ContentSnippet/PartsListElementContentSnippetProvider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Apto\Plugins\PartsListElement\Application\Core\Service\ContentSnippet;
+
+use Apto\Base\Application\Core\Service\ContentSnippet\ContentSnippetProvider;
+use Apto\Base\Application\Core\Service\ContentSnippet\Exceptions\ContentSnippetJsonFileNotFoundException;
+use Apto\Base\Domain\Core\Model\FileSystem\File\File;
+use Apto\Base\Domain\Core\Model\FileSystem\RootFileSystemConnector;
+
+class PartsListElementContentSnippetProvider implements ContentSnippetProvider
+{
+    private RootFileSystemConnector $rootConnector;
+
+    /**
+     * @param RootFileSystemConnector $rootConnector
+     */
+    public function __construct(RootFileSystemConnector $rootConnector)
+    {
+        $this->rootConnector = $rootConnector;
+    }
+
+    /**
+     * @return string
+     * @throws ContentSnippetJsonFileNotFoundException
+     */
+    public function getContentSnippetsJson(): string
+    {
+        $file = File::createFromPath(realpath(__DIR__ . '/../../content-snippet.json'));
```

Comment:
content-snippet.json should be in Resources folder. Have a look at class AptoCatalogContentSnippetProvider for a Path example